src/main/java/zeronote/exceptions/InvalidIndexException.java

```diff
@@ -1,11 +1,16 @@
 package zeronote.exceptions;
 
+//@@Lusi711
```

Comment:
hehe lusi it's supposed to be ``@@author Lusi711`` xD I MADE A MISTAKE.

Reply:
I fixed a few in my PR but others might still be wrong.